### ISSUE TITLE
Bandcamp importer does not work for single release

### ISSUE THREAD

**Issue by rrrnld** [murdos/musicbrainz-userscripts#474]
The user script doesn't run if an artist published only a single release. The script doesn't match the URL and won't run. An example: https://totemcherry.bandcamp.com/releases

I fixed this locally by adding the following line:
``` js
// @include        /^https?://[^/]+/(?:releases)\/?$/
```

I'm not sure if this has any unwanted side effects.

**Comment by liftarn** [comment]
Workaround: Click on one of the track and then "Buy the Full Digital Album" and it would lead you to https://totemcherry.bandcamp.com/album/s-t and the import will work.

**Comment by atj** [comment]
This is specifically what the "[Import Bandcamp releases to MusicBrainz Album Link Helper](https://github.com/murdos/musicbrainz-userscripts#-import-bandcamp-releases-to-musicbrainz-album-link-helper)" userscript is for.

![image](https://user-images.githubusercontent.com/59319/162403036-6cc48e15-275f-46ce-b1f4-527c44dda432.png)

Install it and you will see an "Album page (MB import)" link that will take you to the release:

**Comment by arsinclair** [comment]
Is this still the case? It seems now both https://totemcherry.bandcamp.com/releases and https://totemcherry.bandcamp.com redirect you to the album page. Did Bandcamp fix this? Or does anyone have any examples where problem is still reproduced?

**Comment by rrrnld** [comment]
You're right, it looks like this is solved with the redirect setup by bandcamp. I clicked around and found a couple of other single-release examples, but none where the issue persists:

- https://slitgalleryexhibition.bandcamp.com
- https://thevjjq.bandcamp.com
- https://dorleac.bandcamp.com
